src/transformers/pipelines/pt_utils.py

```diff
@@ -76,14 +76,22 @@ def loader_batch_item(self):
             # Batch data is assumed to be BaseModelOutput (or dict)
             loader_batched = {}
             for k, element in self._loader_batch_data.items():
-                if k in {"hidden_states", "past_key_values", "attentions"} and isinstance(element, tuple):
+                if k in {
+                    "auxiliary_logits",
+                    "hidden_states",
+                    "transformer_decoder_hidden_states",
+                    "pixel_decoder_hidden_states",
+                    "encoder_hidden_states",
+                    "hidden_states",
+                }:
+                    continue
```

Comment:
I would rather not include this code. 

Why does `MaskFormerForInstanceSegmentation` returns those as empty lists instead of not returning them until asked for (like is the default everywhere else I think)

Reply:
Yes this code shouldn't be needed I agree.

Reply:
Not necessary if we do  https://github.com/huggingface/transformers/pull/15936

Reply:
That's the proper fix I agree!